src/components/DocumentationTopic/DocumentationHero.vue

```diff
@@ -153,4 +153,8 @@ $doc-hero-icon-color: dark-color(fill-secondary) !default;
     content: none;
   }
 }
+
+/deep/ a {
```

Comment:
@mportiz08 wont this also change links, when the hero background is disabled?

Reply:
Ah whoops. You're right—I forgot that we aren't always showing it now. I'll make that change.